Tests/OptimizelyTests-Common/DecisionListenerTests.swift

```diff
@@ -778,6 +780,19 @@ class DecisionListenerTests: XCTestCase {
         
         _ = self.optimizely.isFeatureEnabled(featureKey: kFeatureKey, userId: kUserId)
         wait(for: [exp], timeout: 1)
+        
+        let event = getFirstEventJSON()!
+        let visitor = (event["visitors"] as! Array<Dictionary<String, Any>>)[0]
+        let snapshot = (visitor["snapshots"] as! Array<Dictionary<String, Any>>)[0]
+        let decision = (snapshot["decisions"]  as! Array<Dictionary<String, Any>>)[0]
+        
+        let metaData = decision["metadata"] as! Dictionary<String, Any>
+        XCTAssertEqual(metaData["rule_type"] as! String, Constants.DecisionSource.rollout.rawValue)
+        XCTAssertEqual(metaData["rule_key"] as! String, "")
+        XCTAssertEqual(metaData["flag_key"] as! String, "feature_1")
+        XCTAssertEqual(metaData["variation_key"] as! String, "")
+        
+        self.optimizely.config!.project!.sendFlagDecisions = nil
```

Comment:
This test looks good, but looks like in a wrong place, not related to Notification. Can we move this part to other like BathEventTests?